pkg/pillar/cmd/verifier/verifier.go

```diff
@@ -308,9 +309,11 @@ func handleCreate(ctx *verifierContext,
 	status := types.VerifyImageStatus{
 		Name:        config.Name,
 		ImageSha256: config.ImageSha256,
+		MediaType:   config.MediaType,
 		PendingAdd:  true,
 		State:       types.VERIFYING,
 		RefCount:    config.RefCount,
+		MediaTyoe:   config.MediaType,
```

Comment:
It's a typo, I guess?

Reply:
I thought I had fixed that! OK, fixed now.